### PR TITLE
Manticore 0.3.3

### PR DESCRIPTION
**Pull request by ehennenfent**
## 0.3.3 - 2020-01

Thanks to our external contributors!

 - [catenacyber](https://github.com/trailofbits/manticore/commits?author=catenacyber)

### Ethereum
* **[added API]** Flag to only generate alive states when finalizing Manticore [#1554](https://github.com/trailofbits/manticore/pull/1554)
* Fix gas check [#1587](https://github.com/trailofbits/manticore/pull/1587)

### Native
* **[added API]** Add post-instruction hooks [#1579](https://github.com/trailofbits/manticore/pull/1579)

### WASM
* **[added API]** getattr-style calls for WASM functions [#1578](https://github.com/trailofbits/manticore/pull/1578)
* **[changed API]** Pass state to function calls instead of constraint sets [#1578](https://github.com/trailofbits/manticore/pull/1578)

### Other
* **[added API]** Added streamlined state serialization interface [#1596](https://github.com/trailofbits/manticore/pull/1596)
* Fixed Z3 version parsing [#1551](https://github.com/trailofbits/manticore/pull/1551)
* Unique names for ArrayVars [#1552](https://github.com/trailofbits/manticore/pull/1552)
* Improve pickling and multiprocessing compatibility [#1583](https://github.com/trailofbits/manticore/pull/1583)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,29 @@
 
 ## [Unreleased](https://github.com/trailofbits/manticore/compare/0.3.2...HEAD)
```
Comment (ekilmer) [inline CHANGELOG.md]: This line should change too.

**Comment by ehennenfent** [comment]
I'm happy to get #1591, #1589, #1580, #1577, etc pushed out as part of this if we can get them merged by Friday morning.

File: CHANGELOG.md (hunk 1 above)
Comment (ehennenfent) [reply CHANGELOG.md]: Good catch, thanks

File: setup.py [hunk 2]
```diff
@@ -34,13 +34,17 @@ def rtd_dependent_deps():
...
 }
 
+this_directory = os.path.abspath(os.path.dirname(__file__))
+with open(os.path.join(this_directory, "README.md"), encoding="utf-8") as f:
+    long_description = f.read()
 
 setup(
     name="manticore",
     description="Manticore is a symbolic execution tool for analysis of binaries and smart contracts.",
+    long_description=long_description,
```
Comment (ekilmer) [inline setup.py]: Nice add! Was going to open another PR for this, but I forgot

**Pull request merged**